.circleci/config.yml

```diff
@@ -71,6 +71,7 @@ jobs:
           name: "Bootstrapping"
           command: |
             npm run bootstrap
+            npm run build-packs
```

Comment:
Alls tasks are executed in parallel and do not share anything between them.

Inefficient indeed, but fastests and simplest method for now.